specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,214 @@
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request Success Response.",
+            "schema": {
+              "$ref": "#/definitions/metaStoreRequestSuccessResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/metastore/update-database-operations/{id}": {
+      "put": {
+        "tags": [
+          "metastore"
+        ],
+        "operationId": "Metastore_Update",
+        "description": "Update files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/Metastore_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Metadata created Response.",
+            "schema": {
+              "$ref": "#/definitions/metaDataCreatedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/metastore/databases/{id}": {
+      "delete": {
+        "tags": [
+          "metastore"
+        ],
+        "operationId": "Metastore_Delete",
+        "description": "Remove files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/Metastore_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Metadata Deleted Response."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "metaDataCreatedResponse": {
```

Comment:
I did not understand the comment. However, since these are local definitions I think the naming is ok. Please reactivate this comment if you feel otherwise